test/integration/WalletOperations.js

```diff
@@ -117,8 +124,15 @@ describe('Wallet operations', async function() {
     assert.strictEqual(userWalletFactoryResponse.receipt.status, true);
   });
 
-  // wallet3, wallet9 are the owners.
+  // // wallet3, wallet9 are the owners.
+  // wallet1 and wallet2 are the owners.
   it('Should create a user wallet', async function() {
+    await auxiliaryWeb3.eth.accounts.wallet.create(1);
```

Comment:
what s the difference with `await auxiliaryWeb3.eth.accounts.wallet.create(3);` ?

Reply:
@benjaminbollen , Thank you.
Used `await auxiliaryWeb3.eth.accounts.wallet.create(3);` to create 3 accounts.

Reply:
I don't understand the system when we seem to call `await auxiliaryWeb3.eth.accounts.wallet.create` at various places throughout the tests;

this may be less important, and can be addressed later.

Reply:
@benjaminbollen 
By default, Docker geth provides only 3-accounts.
So, to use more accounts,we need to create it dynamically.

Reply:
ok!